### PR TITLE
OpsCenter 6.8.35 Release

### PR DESCRIPTION
**Pull request by vimal-ds**
----
## Release Notes Automation
If you name your pull-request as "Product x.y.z Release ...", after merging the
PR, a GitHub Action will automatically create a product version tag "product-x.y.z".

Supported product names are:
 * DSE
 * OpsCenter
 * Studio
 * Luna Streaming

Version supports 3 sets or 4 sets of digits.

### CODE REVIEW

File: OpsCenter_6.8_Release_Notes.md [hunk 1]
```diff
@@ -1,11 +1,30 @@
...
+* Added {{Materialized View}} restoration progress in the restore dialog. (OPSC-17263)
+
+## Core
+* Fixed potential memory leak in the twisted library. (OPSC-17147)
+
 # Release Notes for OpsCenter 6.8.34
 22 Feb 2024
 
 ## Core
-* Fixed OpsCenter startup failure for customers using a separate storage cluster. Added `alert_on_no_host_available` to the `[storage_cassandra]` section of the cluster conf. When set to `true` an alert will be triggered if *opscenterd* is unable to make a cql connection to the storage cluster. (OPSC-17488)
```
Comment (vimal-ds) [inline OpsCenter_6.8_Release_Notes.md:7]: Modified this based on request from support 
https://datastax.slack.com/archives/C13QK690C/p1710918069041959

File: OpsCenter_6.8_Release_Notes.md [hunk 2]
```diff
@@ -1,11 +1,31 @@
...
-* Added `heartbeat_warn_window` to the `[failover]` section of the `opscenterd.conf`. When set to non zero integer value (seconds), Backup OpsCenter triggers an alert if it does not receive heartbeat from Primary OpsCenter in the given window and added `heartbeat_log_level` to the `[failover]` section of the opscenterd conf. When set to one of these values in `info`, `debug`, `trace` primary and backup OpsCenter logs all the heartbeat communication activities at the set level. (OPSC-17388)
+* Fixed OpsCenter startup failure for customers using a separate storage cluster. Added `alert_on_no_host_available` to the `[storage_cassandra]` section of the cluster conf. When set to `true`  triggers a message in the backup opscenter logs if *opscenterd* is unable to make a cql connection to the storage cluster. (OPSC-17488)
```
Comment (annieden) [inline OpsCenter_6.8_Release_Notes.md:27]:
```suggestion
* Fixed OpsCenter startup failure for customers using a separate storage cluster. Added `alert_on_no_host_available` to the `[storage_cassandra]` section of the cluster configuration. A setting of `true`  triggers a message in the backup opscenter logs when *opscenterd* is unable to make a cql connection to the storage cluster. (OPSC-17488)
```

Comment (annieden) [reply OpsCenter_6.8_Release_Notes.md:27]: Spelling out abbreviation and fixed grammar.

**Review by annieden — APPROVED** [review]
Approved with suggestion applied.

File: OpsCenter_6.8_Release_Notes.md [hunk 3]
```diff
@@ -1,11 +1,31 @@
 # Release notes for OpsCenter
 
+# Release Notes for OpsCenter 6.8.35
+22 Mar 2024
+
+## Backup Service
+* Fixed an issue with PIT restore when there is a schema mismatch. Added {{force_schema}} parameter to PIT Restore API to solve this issue. (OPSC-17486)
```
Comment (msmygit) [inline OpsCenter_6.8_Release_Notes.md:7]: `{{force_schema}}` won't render properly on GitHub markdown files. Instead, start using it with backticket (i.e. the ` character)

Comment (vimal-ds) [reply OpsCenter_6.8_Release_Notes.md:7]: ok

**Pull request merged**